examples/heli 3.mapping

```diff
@@ -0,0 +1,12 @@
+emma_EastPE:aglaope_EastPE
```

Comment:
Duplicate -- it seems like these duplicates might have been created my merge? Please be a little careful when you merge.

Reply:
Removed.